pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -187,6 +187,10 @@ type CertificateSpec struct {
 	// +kubebuilder:validation:ExclusiveMaximum=false
 	// +optional
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"` // Validated by the validating webhook.
+
```

Comment:
nit: wrap lines and add full stop at the ned (since this is exported documentation on the output API/CRD.